feature/bookmarks/src/main/java/com/google/samples/apps/nowinandroid/feature/bookmarks/BookmarksScreen.kt

```diff
@@ -79,41 +109,60 @@ fun BookmarksScreen(
             .fillMaxSize()
             .testTag("bookmarks:feed")
     ) {
+        newsFeed(
+            feedState = feedState,
+            onNewsResourcesCheckedChanged = { id, _ -> removeFromBookmarks(id) },
+        )
+        item(span = { GridItemSpan(maxLineSpan) }) {
+            Spacer(Modifier.windowInsetsBottomHeight(WindowInsets.safeDrawing))
+        }
+    }
+}
 
-        when (feedState) {
-            is NewsFeedUiState.Loading -> {
-                item(span = { GridItemSpan(maxLineSpan) }) {
-                    NiaLoadingWheel(
-                        modifier = Modifier
-                            .fillMaxWidth()
-                            .wrapContentSize()
-                            .testTag("forYou:loading"),
-                        contentDesc = stringResource(id = R.string.saved_loading),
-                    )
-                }
-            }
+@Composable
+private fun EmptyState(modifier: Modifier = Modifier) {
+    Column(
+        modifier = modifier
+            .padding(16.dp)
+            .fillMaxSize()
+            .testTag("bookmarks:empty"),
+        verticalArrangement = Arrangement.Center,
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
+        Image(
+            modifier = Modifier.fillMaxWidth(),
+            painter = painterResource(id = R.drawable.img_empty_bookmarks),
+            contentDescription = null
+        )
 
-            is NewsFeedUiState.Success -> {
-                if (feedState.feed.isNotEmpty()) {
-                    newsFeed(
-                        feedState = feedState,
-                        onNewsResourcesCheckedChanged = { id, _ -> removeFromBookmarks(id) },
-                    )
-                    item(span = { GridItemSpan(maxLineSpan) }) {
-                        Spacer(Modifier.windowInsetsBottomHeight(WindowInsets.safeDrawing))
-                    }
-                } else item(span = { GridItemSpan(maxLineSpan) }) {
-                    Box(
-                        modifier = Modifier
-                            .fillMaxHeight()
-                            .wrapContentSize()
-                            .testTag("bookmarks:empty"),
-                        contentAlignment = Alignment.Center
-                    ) {
-                        Text(text = stringResource(id = R.string.bookmarks_empty))
-                    }
-                }
-            }
-        }
+        Spacer(modifier = Modifier.height(16.dp))
+
+        Text(
+            text = stringResource(id = R.string.bookmarks_empty_error),
+            modifier = Modifier.fillMaxWidth(),
+            textAlign = TextAlign.Center,
+            style = MaterialTheme.typography.titleMedium,
+            fontWeight = FontWeight.Bold
+        )
+
+        Spacer(modifier = Modifier.height(5.dp))
```

Comment:
this spacing is a bit odd. Won't `Arrangement.spacedBy` be a better alternative?

Reply:
Agreed that the Spacer makes more sense here, although in Figma it seems to be 8dp, not 5dp.

Reply:
Good point! Just changed it from 5 to 8dp. Please check the latest commit.